netket/graph/common_lattices.py

```diff
@@ -78,6 +87,8 @@ def Grid(
              This parameter can also be a list of booleans with same length as
              the parameter `length`, in which case each dimension will have
              PBC/OBC depending on the corresponding entry of `pbc`.
+        color_edges: generates nearest-neighbour edges colored according to direction
+                     cannot be used with `max_neighbor_order` or `custom_edges`
```

Comment:
Can you better rephrase this docstring? Does this mean that bonds along the X direction have a different color of bonds along the Y directions?

Maybe just an example can clairfy, and specifying then how many colors to expect. Also write (default=False) at the end.

Reply:
How about this?
```suggestion
        color_edges: generates nearest-neighbour edges colored according to direction
                     i.e. edges along Cartesian direction #i have color i
                     cannot be used with `max_neighbor_order` or `custom_edges`
```